truco_mobile/lib/src/model/card_model.dart

```diff
@@ -0,0 +1,23 @@
+class Card {
+  final String rank;
+  final String suit;
+  final int value;
```

Comment:
Nesse contexto também teria que analisar pois os `final String value` significa que a propiedade value só pode ser preenchida uma vez e nesse contexto acho que não tem nessecidade de usar o final pois o value de uma carta pode variar, ex: se for uma manilha

Reply:
Sim essa questão da manilha ainão não  consegui ver como fazer, mas ai é só retirar a declaração do Final